packages/ember-self-focused/package.json

```diff
@@ -30,6 +30,7 @@
     "ember-ajax": "^3.0.0",
     "ember-cli": "~3.2.0",
     "ember-cli-dependency-checker": "^2.0.0",
+    "ember-cli-fastboot": "^2.0.4",
```

Comment:
Shouldn't this be regular dependencies?

Reply:
Hmm, not sure. Dinesh suggested it to a `devDependency` also I used `ember-cli i ember-cli-fastboot` which added it as `devDependency`.

Reply:
Right you are, sorry I missed that